Structure_Engine/Query/EmbodiedCarbon.cs

```diff
@@ -30,13 +34,19 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Calculates the total amount of embodied carbon of a Bar by taken the mass of the bar (as section area*length*density) times the EmbodiedCarbon value of the material")]
```

Comment:
Calculates the total amount of embodied carbon of a Bar by getting the mass of the bar (as section area*length*density) multiplied by the EmbodiedCarbon value of the material